website/docs/r/wiki.html.markdown

```diff
@@ -39,6 +39,13 @@ resource "azuredevops_wiki" "example2" {
   type          = "codeWiki"
```

Comment:
please add a new wiki page document for this new resource.

Reply:
Sure, let me study the document format first.
I'll submit this part later. Thanks